README.md

```diff
@@ -93,3 +93,109 @@ allow_project_configs=true
 org=my_okta_org
 user=my_user
 ```
+
+## How to use bmx and the AWS cli 🚲 - Opinionated Workflow
+
+### Pre-requisites
+- [Install the aws cli](https://aws.amazon.com/cli/)
+- Install and configure the d2l bmx tool
+  - [bmx repo](https://github.com/Brightspace/bmx)
+  - [Confluence guide on how to install bmx](https://desire2learn.atlassian.net/wiki/spaces/TH/pages/1011745036/Installing+and+using+BMX)
```

Comment:
Consider linking to https://github.com/Brightspace/bmx/wiki/Quick-Start , its more recently updated. Wasn't really aware of that confluence page at all TBH

Reply:
Linked to the wiki instead 👍 
https://github.com/Brightspace/bmx/pull/212/commits/aa54f4207b26faabcdb33d4d62d7678b0a9e3141